Code/GraphMol/Wrap/rdchem.cpp

```diff
@@ -98,6 +102,8 @@ T *next_ptr(O &self) {
 BOOST_PYTHON_MODULE(rdchem) {
   python::scope().attr("__doc__") =
       "Module containing the core chemistry functionality of the RDKit";
+  const bool register_scalar_converters=false;
+  boost::python::numpy::initialize(register_scalar_converters);
```

Comment:
It might only be necessary if we are creating ndarray's not accepting them.  I was following the boiler plate here:

https://www.boost.org/doc/libs/1_84_0/libs/python/doc/html/numpy/tutorial/ndarray.html

Reply:
I can try a build with it gone and see if it still works.

Reply:
It does look like it's required:  https://stackoverflow.com/questions/54904448/boost-python-nullptr-while-extracting-ndarray

Reply:
verified: It segfaults if we don't initialize.